src/pages/getting-started/index.mdx

```diff
@@ -38,121 +62,106 @@ From a more tech perspective, the LiveChat Platform provides:
+- <a href="https://developers.livechat.com/docs/extending-chat-widget#javascript-api" target="_blank">JavaScript API</a> changing the look and behavior of the Chat Widget.
+- <a href="https://developers.livechat.com/docs/extending-chat-widget#moments" target="_blank">Moments</a> allowing customers to perform specific actions.
+- <a href="https://developers.livechat.com/docs/extending-chat-widget#rich-messages" target="_blank">Rich messages</a> making your messages more stylish and interactive.
+- <a href="https://developers.livechat.com/docs/extending-chat-widget#customer-sdk" target="_blank">Customer SDK</a> allowing you to build a chat widget from scratch.
+- <a href="https://developers.livechat.com/docs/extending-chat-widget#custom-identity-provider" target="_blank">Custom Identity Provider</a> giving you the possibility of creating a cross-platform communication.
+- <a href="https://developers.livechat.com/docs/extending-chat-widget#chat-widget-adapters" target="_blank">Chat Widget Adapters</a> allowing you to use the Chat Widget with popular frontend frameworks.
 
-# HelpDesk and LiveChat integration
+# Monetization
 
-You can integrate LiveChat with HelpDesk to get a powerful tool helping you to manage all your communication with your customers in one place. 
+Platform provides you with <a href="https://developers.livechat.com/docs/monetization" target="_blank">monetization tools</a> allowing you to start getting profit from the apps you have created. All the payments are processed within Text eco-system using in-app payments. Thanks to this solution, you have more precise control over monetizing features of your app. Publish your apps and integrations on our <Placeholder id="LIVECHAT_MARKETPLACE_URL"/> or <a href="https://www.helpdesk.com/marketplace/" target="_blank">HelpDesk Marketplace</a> and provide people with the necessary solutions to meet their needs.
 
-<a href="https://www.helpdesk.com/help/livechat-integration-manage-tickets-and-chats-in-one-application/" target="_blank">Read more</a> about HelpDesk and LiveChat integration or buy our service directly on the <a href="https://www.livechat.com/marketplace/apps/helpdesk/#description" target="_blank">Marketplace</a>.
+Get familiar with the <a href="https://developers.livechat.com/docs/monetization/app-review-process" target="_blank">app review process</a> and read about good practices and rules that may help you improve your app and make it more attractive for the customers. Decide to go with the <a href="https://developers.livechat.com/docs/monetization/app-review-process#basic-app-review" target="_blank">Basic app review</a> or <a href="https://developers.livechat.com/docs/monetization/app-review-process#premium-app-review" target="_blank">Premium app review</a>. Premium app review provides detailed technical, design, and copy reviews with feedback or suggestions on improving your product.
```

Comment:
I don't think we need to go into details about the difference between basic and paid app review. The information that app goes thorough an ap review process before publishing is enough, IMO

Reply:
Yup, I agree on this one.